Tests/Recurly/Invoice_Test.php

```diff
@@ -111,4 +111,12 @@ public function testGetInvoicePdf() {
     $result = Recurly_Invoice::getInvoicePdf('abcdef1234567890', 'en-GB', $this->client);
     $this->assertEquals(array('/invoices/abcdef1234567890', 'en-GB'), $result);
   }
+
+  public function testRefundAmount() {
+    $this->client->addResponse('POST', 'https://api.recurly.com/v2/invoices/abcdef1234567890/refund', 'invoices/refund-201.xml');
+    $invoice = Recurly_Invoice::get('abcdef1234567890', $this->client);
```

Comment:
i think this will conflict with barton's change that i just merged. i think this needs to switch this to 1000.